webhook/certificates/certificates.go

```diff
@@ -28,6 +31,11 @@ import (
 	certresources "knative.dev/pkg/webhook/certificates/resources"
 )
 
+const (
+	// Time used for updating a certificate before it expires. Default 1 week.
+	oneWeek = 168 * time.Hour
```

Comment:
`default one week` is kind of strange here, since it's a constant.
I'd also write `7*24*time.hour`, which is more expressive.

Reply:
Makes sense, so remove the comment about default and change from 168 to 7*24?

Reply:
I think leaving the comment is fine, but now it's more clearly accurate :D